addons/hw_drivers/tools/iot_system.py

```diff
@@ -0,0 +1,46 @@
+
+from enum import Enum, unique
+from functools import cache
+import logging
+import platform
+
+_logger = logging.getLogger(__name__)
+
+
+@unique
+class IoTSystem(Enum):
+    TEST = 'T'
+    IOT_BOX = 'L'
+    WINDOWS = 'W'
```

Comment:
IoT Box is used for both "VIrtual" and "Raspberry Pi" IoT boxes
```suggestion
    RPI = 'L'
    WINDOWS = 'W'
```